values.schema.json

```diff
@@ -837,6 +837,27 @@
                         "additionalProperties": false
                     },
                     "description": "A list of MySQL users and their privileges."
+                },
+                "config": {
+                    "type": "object",
+                    "description": "MySQL configuration options",
+                    "properties": {
+                        "maxConnections": {
+                            "type": "integer",
+                            "description": "Maximum number of connections to the MySQL server.",
+                            "default": 512
+                        },
+                        "maxConnectErrors": {
+                            "type": "integer",
```

Comment:
If the size of the number is problematic, prefer using string. We cannot tolerate a rounded float in this configuration option.